src/views/artworkDetails.js

```diff
@@ -22,29 +22,51 @@ export default function ArtworkDetails({navigation}) {
     quantity: 1,
   };
 
-   let arr;
+  var arr;
   const getData = async () => {
     try {
       await AsyncStorage.getItem('save_data')
```

Comment:
You have defined getData in AsyncStorageService, then directly use that function to get jsonValue from asyncStorage